api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -441,11 +441,44 @@ public void validate_shouldPassValidationIfFieldLengthsAreCorrect() {
 	@Test
 	public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() {
 		Visit visit = makeVisit(42);
-		visit
-		        .setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
-		
+		visit.setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
```

Comment:
From the description of the ticket, I concluded that it just required tests to check that the start date is not before the birth date. I may be wrong though because I didnt really think of changing the implementation. Could you please take a look at the ticket description [https://issues.openmrs.org/browse/TRUNK-4939?filter=-1](url) and tell me what you think?